screens/wishlist/wishlist.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./wishlist.css">
+    <title>Wishlist</title>
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="nav bg-radient">
+        <div class="nav-logo">
+            <span class="nav-header l-sp-1">Pure<span class="txt-theme txt-bold">Fitness</span></span>
```

Comment:
try to use descriptive classnames, here its not clear what does the `l-sp-1` class do

Reply:
l-sp is for letter spacing